cmd/containers-storage/main.go

```diff
@@ -84,63 +85,61 @@ func main() {
 	cmd := args[0]
 
 	for _, command := range commands {
```

Comment:
Non-blocking: This works, and it is an improvement already, but processing `commands` in a loop is not quite what we _mean_.

Consider
```go
i := slices.IndexFunc(commands, func(c) { slices.Contains(c.names, cmd) })
if i == -1 { /* fail */ }
command := commands[i]
// straight-line top-level code processing the command we found
```

Reply:
You are right, this looks weird.

Rewrote (I left `for range` as it seems more readable than `slices.IndexFunc` in this particular case).